lib/SPIRV/SPIRVReader.cpp

```diff
@@ -450,26 +450,28 @@ Type *SPIRVToLLVM::transType(SPIRVType *T, bool UseTPT) {
       Params.push_back(static_cast<SPIRVConstant *>(Use)->getZExtIntValue());
     auto *CTI = MT->getComponentTypeInterpretation();
     if (!CTI)
-      return mapType(T, getSPIRVType(internal::OpTypeJointMatrixINTEL,
-                                     transTypeToOCLTypeName(MT->getCompType()),
-                                     Params, !UseTPT));
-    std::string ComponentTypeName;
+      return mapType(
+          T, llvm::TargetExtType::get(*Context, "spirv.JointMatrixINTEL",
+                                      transType(MT->getCompType()), Params));
     switch (static_cast<SPIRVConstant *>(CTI)->getZExtIntValue()) {
     case internal::InternalJointMatrixCTI::TF32:
-      ComponentTypeName = "tf32";
+      Params.push_back(1);
```

Comment:
Do we have some enum for these values?

Reply:
Oh, we do have them :) it is "internal::InternalJointMatrixCTI::"